models/lawnmower/model.sdf

```diff
@@ -0,0 +1,582 @@
+            <ode>
+              <mu>0</mu>
+              <mu2>0</mu2>
+              <slip1>1.0</slip1>
+              <slip2>1.0</slip2>
+            </ode>
+          </friction>
+        </surface>
+      </collision>
+      <visual name="left_caster_visual_base">
+        <pose>0.85 0.38 -0.01 0 0 0</pose>
+        <geometry>
+          <cylinder>
+            <radius>0.1</radius>
+            <length>0.1</length>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.9 0.36 0 1</ambient>
+          <diffuse>0.9 0.36 0 1</diffuse>
+          <specular>0 0 0 0</specular>
+          <emissive>0 0 0 1</emissive>
+        </material>
+      </visual>
+      <visual name="left_caster_visual">
+        <pose>0.85 0.38 -0.1 0 0 0</pose>
+        <geometry>
+          <sphere>
+            <radius>.1</radius>
+          </sphere>
+        </geometry>
+        <material>
+          <ambient>1.0 0.2 0.2 1</ambient>
+          <diffuse>1.0 0.1 0.1 1</diffuse>
+          <specular>0 0 0 0</specular>
+          <emissive>0 0 0 1</emissive>
+        </material>
+      </visual>
+      <collision name="right_caster_collision">
+        <pose>0.85 -0.38 -0.1 0 0 0</pose>
+        <geometry>
+          <sphere>
+            <radius>.1</radius>
+          </sphere>
+        </geometry>
+        <surface>
+          <friction>
+            <ode>
+              <mu>0</mu>
+              <mu2>0</mu2>
+              <slip1>1.0</slip1>
+              <slip2>1.0</slip2>
+            </ode>
+          </friction>
+        </surface>
+      </collision>
+      <visual name="right_caster_visual_base">
+        <pose>0.85 -0.38 -0.01 0 0 0</pose>
+        <geometry>
+          <cylinder>
+            <radius>0.1</radius>
+            <length>0.1</length>
+          </cylinder>
+        </geometry>
+        <material>
+          <ambient>0.9 0.36 0 1</ambient>
+          <diffuse>0.9 0.36 0 1</diffuse>
+          <specular>0 0 0 0</specular>
+          <emissive>0 0 0 1</emissive>
+        </material>
+      </visual>
+      <visual name="right_caster_visual">
+        <pose>0.85 -0.38 -0.1 0 0 0</pose>
+        <geometry>
+          <sphere>
+            <radius>.1</radius>
+          </sphere>
+        </geometry>
+        <material>
+          <ambient>0.2 1.0 0.2 1</ambient>
+          <diffuse>0.1 1.0 0.1 1</diffuse>
+          <specular>0 0 0 0</specular>
+          <emissive>0 0 0 1</emissive>
+        </material>
+      </visual>
+      <sensor name="air_pressure_sensor" type="air_pressure">
+        <always_on>1</always_on>
+        <update_rate>50</update_rate>
+        <air_pressure>
+          <pressure>
+            <noise type="gaussian">
+              <mean>0</mean>
+              <stddev>0.01</stddev>
+            </noise>
+          </pressure>
+        </air_pressure>
+      </sensor>
+      <sensor name="imu_sensor" type="imu">
+        <always_on>1</always_on>
+        <update_rate>250</update_rate>
+      </sensor>
```

Comment:
Can we add the navsat sensor here? There is currently a push to ensure that all models have accurate GPS positioning within a world. For this to work it needs to have the navsat sensor. This can be added as:
```suggestion
      </sensor>
      <sensor name="navsat_sensor" type="navsat">
        <always_on>1</always_on>
        <update_rate>30</update_rate>
      </sensor>
```

Reply:
@frede791 - I see some references to "**navsat**" in main (_ubx.cpp_ and _ubx.h_), but none in existing models. Would you mind bringing up your suggestion in the Mainteiner's Discord today?
BTW, the _air_pressure_sensor_ is, probably, not needed here, although it is present in R1 rover model (@PerFrivik - what do you think?)

Reply:
@frede791 @PerFrivik  - I experimented with:
1. Removing air_pressure_sensor causes magnitometer failure and other strange effects, so it has to stay.
2. Adding navsat_sensor (as you suggested) seems to improve position reporting in QGC, and no negative effects, so I will be adding it - thanks!

I am not sure how it interacts with EKF2 for position reporting - but my interest is purely academical at this point.

Reply:
@slgrobotics I'm currently working on a PR that aims to add this: https://github.com/PX4/PX4-gazebo-models/pull/26

Reply:
https://github.com/PX4/PX4-Autopilot/pull/22638  - a PR about that **navsat_sensor**, for reference

Reply:
@frede791 - got it, thanks! I added your suggestion "as a batch" (not sure it was right thing to do).

I don't have merging rights across PX4, so somebody has to do that when we are all done with the review process.